x-pack/plugins/spaces/public/management/components/enabled_features/enabled_features.tsx

```diff
@@ -40,7 +40,7 @@ export const EnabledFeatures: FunctionComponent<Props> = (props) => {
             <p>
               <FormattedMessage
                 id="xpack.spaces.management.enabledSpaceFeatures.chooseFeaturesToDisplayMessage"
-                defaultMessage="Choose the features to display in the navigation menu for users of this space. If you want to focus on a single solution, you can simplify the navigation even more by selecting a {solutionView}."
+                defaultMessage="Choose the features to display in the navigation menu for users of this space. If you want to focus on a single solution, you can simplify the navigation even more by selecting a solutionView."
```

Comment:
Were the curly braces needed? it says `solutionView` on the UI now:
<img width="1271" alt="Screenshot 2024-10-22 at 2 21 21 PM" src="https://github.com/user-attachments/assets/518aece6-f934-4afe-8a06-3ee72e028e61">

Reply:
Oh yes, sorry my bad. I should have just removed `<strong>` tag.
![CleanShot 2024-10-23 at 08 50 18@2x](https://github.com/user-attachments/assets/27f6068d-dbe0-4681-afab-dfd856969775)